src/test/java/com/wikia/webdriver/elements/mercury/components/RecentWikiActivity.java

```diff
@@ -0,0 +1,151 @@
+package com.wikia.webdriver.elements.mercury.components;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+
+public class RecentWikiActivity extends WikiBasePageObject {
+
+  @FindBy(css = ".recent-change__see-diff")
+  private WebElement seeDiffArrow;
+
+  @FindBy(css = ".diff-page__undo")
+  private WebElement undoButton;
+
+  @FindBy(css = "button.sub-head--cancel")
+  private WebElement backButton;
+
+  @FindBy(css = ".sub-head--cancel")
+  private WebElement backArrow;
+
+  @FindBy(css = ".sub-head--done")
+  private WebElement undoButtonOnSummaryPage;
+
+  @FindBy(css = ".confirmation-message__textarea")
+  private WebElement summaryTestbox;
+
+  @FindBy(css = ".recent-change")
+  private WebElement diff;
+
+  @FindBy(css = ".alert-notification.alert-box.success")
+  private WebElement successNote;
+
+  private By undoButtonBy = By.cssSelector(".diff-page__undo");
+
+  private Loading loading;
+  private Actions actions;
+
+  public RecentWikiActivity() {
+    super();
+
+    this.loading = new Loading(driver);
+    this.actions = new Actions(driver);
+  }
+
+  public RecentWikiActivity undoButtonNotVisible() {
+    wait.forElementNotVisible(undoButtonBy);
+    PageObjectLogging.logInfo("Undo button is NOT visible");
+
+    return this;
+  }
+
+  public RecentWikiActivity openDiffPage() {
+    wait.forElementClickable(seeDiffArrow);
+
+    String urlToBeRedirected = seeDiffArrow.getAttribute("href");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "urlToBeRedirected" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)